tpp-prophets/peptide_prophet.xml

```diff
@@ -18,92 +17,66 @@
+        <param argument="--usepi" type="boolean" label="Use pI information" truevalue="--usepi" falsevalue=""/>
+        <param argument="--usert" type="boolean" label="Use hydrophobicity / RT information" truevalue="--usert" falsevalue=""/>
+        <param argument="--accurate-mass" name="accurate_mass" type="boolean" label="Use accurate mass binning" truevalue="--accurate-mass" falsevalue=""/>
+        <param argument="--no-ntt" name="no_ntt" type="boolean" label="Don't use NTT model" truevalue="--no-ntt" falsevalue=""/>
+        <param argument="--no-nmc" name="no_nmc" type="boolean" label="Don't use NMC model" truevalue="--no-nmc" falsevalue=""/>
+        <param argument="--usegamma" type="boolean" label="Use Gamma distribution to model the negatives" help="Applies only to X!Tandem results" truevalue="--usegamma" falsevalue=""/>
+        <param argument="--use-only-expect" name="use_only_expect" type="boolean" label="Only use Expect Score as the discriminant" help="Applies only to X!Tandem results. 
             Helpful for data with homologous top hits e.g. phospho or glyco" truevalue="--use-only-expect" falsevalue=""/>
         <param name="force_fit" type="boolean" label="Force fitting" help="Bypasses automatic mixture model checks and forces fitting of a mixture model" truevalue="--force-fit" falsevalue=""/>
         <param name="allow_alt_instruments" type="boolean" label="Allow multiple instrument types" help="Warning instead of exit with error if instrument types between runs is different" truevalue="--allow-alt-instruments" falsevalue=""/>
         <param name="maldi" type="boolean" label="Maldi data" truevalue="-l" falsevalue=""/>
         <param name="usedecoys" type="boolean" label="Use decoys to pin down the negative distribution" truevalue="" falsevalue="--no-decoy"/>
-        <param name="decoy_prefix_string" help="Prefix string for decoy ids" type="text" value="decoy_" label="Decoy Prefix String" size="20"/>    
-        <param name="experiment_label" help="Used to commonly label all spectra from one experiment" type="text" value="" label="Experiment Label" size="20"/>
+        <param name="decoy_prefix_string" help="Prefix string for decoy ids" type="text" value="decoy_" label="Decoy Prefix String"/>
+        <param name="experiment_label" help="Used to commonly label all spectra from one experiment" type="text" value="" label="Experiment Label"/>
         <param name="p_thresh" help="Peptides scoring less than this value are discarded" type="float" value="0.05" min="0" max="1" label="Probability Threshold"/>
     </inputs>
     <outputs>
-        <data format="peptideprophet_pepxml" name="output" metadata_source="input_file" label="peptide_prophet.${input_file.display_name}.pep.xml" from_work_dir="peptide_prophet_output.pep.xml"/>
+        <data name="output" from_work_dir="peptide_prophet_output.pep.xml" format="peptideprophet_pepxml"/>
```

Comment:
Why did you remove the stuff about setting the label? And are you sure it will work now without the from_work_dir?

Reply:
from_work_dir is still there, I just removed the label, but can bring it back. Galaxy has "label" as build in feature nowadays and this might help to archive what you wanted to do here. But I can bring it back.

Reply:
Yes please bring back the label.